### DISCUSSION TITLE
Client can't get IP address (app stopped working)

### DISCUSSION THREAD

**Discussion by forestfoxx** [Mygod/VPNHotspot#454] [Q&A]
Been running 2.13 (I think) for a while just fine (Android 8.1.0), it was working just a few days ago. Yet now I can't get it to work for some reason, the client (Windows/Linux) connected by usb (I never used other options) can't seem to get the IP address, judging by the notification I get. The app shows that "0 devices connected" when it usually was 1. Tried to enable DHCP workaround, no change. Updated to 2.15.4, also no change. Tried another VPn, didn't help. Scratching my head here since I haven't changed any network settings for a while, and my VPN works just fine too.

**Comment by Mygod** [comment]
Did you update anything (Android system etc)? Does system USB tethering (without using this app) work?

**Reply by forestfoxx** [reply]
No, same Android version, kernel version, etc. Maybe some google connectivity services, can't recall...updated them now just in case, no change. "IP configuration unavailable".

System USB tethering seems to lead to the exact same mistake and is not working. I haven't used it before though, went straight for the VPN, so nothing to compare with :\

**Reply by forestfoxx** [reply]
A simple phone reboot solved it. Dunno what was the issue, though
